aws/terraform/eks_cluster.go

```diff
@@ -50,6 +50,9 @@ func decodeEKSClusterValues(tfVals map[string]interface{}) (eKSClusterValues, er
 
 // NewInstance creates a new Instance from Terraform values.
 func (p *Provider) newEKSCluster(vals eKSClusterValues) *EKSCluster {
+	// Currently, it is not utilized in this resource.
+	_ = vals
```

Comment:
There is no need to do this, GO will not complain, but if you want to make it explicit is better to do on the actual signature so `newEKSCluster( _ eKSClusterValues)`.

But again it's not required.

Reply:
Yes this is more an editor extension issue to avoid `unused parameter: vals` warning.
I will update the signature